db/src/main/java/io/lantern/db/DB.kt

```diff
@@ -259,7 +259,7 @@ class DB private constructor(
         // Create a table for full text search
         db.execSQL(
             """CREATE VIRTUAL TABLE IF NOT EXISTS ${schema}_fts
-                        USING fts5(value, content=data, tokenize='porter unicode61')"""
+                        USING fts5(value, tokenize='porter trigram')"""
```

Comment:
Instead of using an external content table, we're using a regular fts5 table now.

To get prefix matching and highlighting, we need to use the experimental `trigram` tokenizer.